### DISCUSSION TITLE
CUDA 12.4

### DISCUSSION THREAD

**Discussion by yamatazen** [lllyasviel/stable-diffusion-webui-forge#989] [Q&A] [answered]
What GPUs are supported by CUDA 12.4?

**Comment by Hapseleg** [comment]
https://letmegooglethat.com/?q=nvidia+cuda+supported+gpu

**Comment by Susanoo1337** [comment, marked as answer]
Don't we all wish we could just swipe on a bad response?

Anyway, any GPU with CUDA Compute of 5.0 or higher, so all the way back to the GTX 750/Ti.
(Edit: actually let me specify, all Maxwell architecture GTX cards, I realize that most of the GTX 7xx series was on Kepler which had a lower compute.)
